cmd/dep/status.go

```diff
@@ -191,6 +192,25 @@ func (out *dotOutput) MissingHeader()                {}
 func (out *dotOutput) MissingLine(ms *MissingStatus) {}
 func (out *dotOutput) MissingFooter()                {}
 
+type templateOutput struct {
+	w    io.Writer
+	tmpl *template.Template
+}
+
+func (out *templateOutput) BasicHeader() {}
+func (out *templateOutput) BasicFooter() {}
+
+func (out *templateOutput) BasicLine(bs *BasicStatus) {
+	out.tmpl.Execute(out.w, bs)
```

Comment:
Should errors be captured? Don't necessarily need to halt on error since it may not affect all lines (although `go list -f` does halt), but I'm not sure whether the feedback would be better in-line to `out.w` or routed to `ctx.Out/Err`. Maybe `Err`, to cleanly separate it for anything parsing.

Reply:
oh!  I didn't realize that it returned an error. Maybe because we are not handling the returned error in any of the other writes, like error from `fmt.Fprintf()` in `tableOutput` and `json.Encode()` in `jsonOutput`. Encoding failure seems important to handle to me.
Routing the error to `Err` should be better; similar to how we keep the partial status output separate from the raised errors.

Reply:
@darkowlzz so is there a change to be made here? this was on my list to "merge fast, fail fast," but it seems like you're planning on a followup.